src/test/java/uk/gov/digital/ho/pttg/application/namematching/candidates/SpecialCharactersTest.java

```diff
@@ -0,0 +1,181 @@
+package uk.gov.digital.ho.pttg.application.namematching.candidates;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.digital.ho.pttg.application.namematching.CandidateName;
+import uk.gov.digital.ho.pttg.application.namematching.InputNames;
+
+import java.util.List;
+
+import static java.util.Collections.emptyList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class SpecialCharactersTest {
```

Comment:
Should we have one test with multiple types of special characters to prove they work in conjunction?

Reply:
Yes - I'll add that